src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -376,10 +384,12 @@ def update_container_settings(resource_group_name, name, docker_registry_server_
         settings.append('DOCKER_REGISTRY_SERVER_USERNAME=' + docker_registry_server_user)
     if docker_registry_server_password is not None:
         settings.append('DOCKER_REGISTRY_SERVER_PASSWORD=' + docker_registry_server_password)
+    if websites_enable_app_service_storage is not None:
```

Comment:
The condition test is inconsistent with the one on line 392. Note that `websites_enable_app_service_storage is not None` is not equivalent to `bool(websites_enable_app_service_storage)`. When `websites_enable_app_service_storage == ''`, the former is **true** while the latter is **false**.

Reply:
Well websites_enable_app_service_storage has to be either true or false (as defined in _params.py). How about if I just do a websites_enable_app_service_storage = str(websites_enable_app_service_storage) at the top of this function? This way bool('false') -> true

Reply:
Ok, so this parameter can be 'True', 'False', 'None'. Here you only need to test:
```
if websites_enable_app_service_storage
```
Some supplement reading: https://www.udacity.com/wiki/cs258/truthiness-in-python.